src/pages/CreateCirclePage/CreateCirclePage.tsx

```diff
@@ -0,0 +1,263 @@
+
+  const protocols = useMemo(
+    () =>
+      uniqBy(
+        myAdminCircles.map(({ protocol }) => protocol),
+        'id'
+      ),
+    [myAdminCircles]
+  );
+
+  const { createCircle } = useApi();
+
+  if (myAddress === undefined) {
+    return <></>;
+  }
+
+  return (
+    <div className={classes.root}>
+      <h2 className={classes.title}>Create a Circle</h2>
+      <CreateCircleForm.FormController
+        source={undefined}
+        hideFieldErrors
+        submit={({
+          h_captcha_token,
+          research_what_needs,
+          research_how_use,
+          research_org_structure,
+          research_org_link,
+          research_org_ens,
+          research_contact,
+          ...params
+        }) =>
+          createCircle(
+            { address: myAddress, ...params },
+            h_captcha_token,
+            JSON.stringify({
+              address: myAddress,
+              research_what_needs,
+              research_how_use,
+              research_org_structure,
+              research_org_link,
+              research_org_ens,
+              research_contact,
+              ...params,
+            })
+          )
+        }
+      >
+        {({ fields, errors, changedOutput, handleSubmit }) => (
+          <div className={classes.bodyInner}>
+            <div className={classes.subtitle}>
+              Coordinape circles allow you to collectively reward circle members
+              through equitable and transparent payments. To start a circle, we
+              need just a bit of information.
+            </div>
+            <div className={classes.twoColumnGrid}>
+              <FormTextField {...fields.user_name} label="Username" fullWidth />
+              <ApeTextField
+                label="ETH Address"
+                value={myAddress}
+                disabled
+                fullWidth
+              />
+              {myAdminCircles ? (
+                <FormAutocomplete
+                  {...fields.protocol_name}
+                  value={fields.protocol_name.value}
+                  onChange={(v: string) => {
+                    const id = protocols.find((p) => p.name === v)?.id;
+                    fields.protocol_id?.onChange(id);
+                    fields.protocol_name.onChange(v);
+                  }}
+                  options={protocols.map((p) => p.name)}
+                  label="Organization Name"
+                  helperText="Select from list to add to existing org."
+                  fullWidth
+                />
+              ) : (
+                <FormTextField
+                  {...fields.protocol_name}
+                  label="Organization Name"
+                  fullWidth
+                />
+              )}
+              <FormTextField
+                {...fields.circle_name}
+                label="Circle Name"
+                fullWidth
+              />
+            </div>
+            <h3 className={classes.sectionTitle}>Optional Fields</h3>
```

Comment:
if the user's already a circle admin, should we skip all the research fields?

Reply:
It's not so intense now..
I am not opposed to this, just don't feel strongly to do it